src/components/AddressSearch/index.tsx

```diff
@@ -1,184 +1,77 @@
-                    /** Lattitude of the location */
-                    lat: PropTypes.number,
-
-                    /** Longitude of the location */
-                    lng: PropTypes.number,
-                }),
-            }),
-        }),
-    ),
-
-    /** A map of inputID key names */
-    renamedInputKeys: PropTypes.shape({
-        street: PropTypes.string,
-        street2: PropTypes.string,
-        city: PropTypes.string,
-        state: PropTypes.string,
-        lat: PropTypes.string,
-        lng: PropTypes.string,
-        zipCode: PropTypes.string,
-    }),
-
-    /** Maximum number of characters allowed in search input */
-    maxInputLength: PropTypes.number,
-
-    /** The result types to return from the Google Places Autocomplete request */
-    resultTypes: PropTypes.string,
-
-    /** Information about the network */
-    network: networkPropTypes.isRequired,
-
-    /** Location bias for querying search results. */
-    locationBias: PropTypes.string,
-
-    ...withLocalizePropTypes,
-};
-
-const defaultProps = {
-    inputID: undefined,
-    shouldSaveDraft: false,
-    onBlur: () => {},
-    onPress: () => {},
-    errorText: '',
-    hint: '',
-    value: undefined,
-    defaultValue: undefined,
-    containerStyles: [],
-    isLimitedToUSA: false,
-    canUseCurrentLocation: false,
-    renamedInputKeys: {
-        street: 'addressStreet',
-        street2: 'addressStreet2',
-        city: 'addressCity',
-        state: 'addressState',
-        zipCode: 'addressZipCode',
-        lat: 'addressLat',
-        lng: 'addressLng',
-    },
-    maxInputLength: undefined,
-    predefinedPlaces: [],
-    resultTypes: 'address',
-    locationBias: undefined,
-};
-
-function AddressSearch({
-    canUseCurrentLocation,
-    containerStyles,
-    defaultValue,
-    errorText,
-    hint,
-    innerRef,
-    inputID,
-    isLimitedToUSA,
-    label,
-    maxInputLength,
-    network,
-    onBlur,
-    onInputChange,
-    onPress,
-    predefinedPlaces,
-    preferredLocale,
-    renamedInputKeys,
-    resultTypes,
-    shouldSaveDraft,
-    translate,
-    value,
-    locationBias,
-}) {
+function AddressSearch(
+    {
+        canUseCurrentLocation = false,
+        containerStyles,
+        defaultValue,
+        errorText = '',
+        hint = '',
+        inputID,
+        isLimitedToUSA = false,
+        label,
+        maxInputLength,
+        onBlur,
+        onInputChange,
+        onPress,
+        predefinedPlaces = [],
+        preferredLocale,
+        renamedInputKeys = {
+            street: 'addressStreet',
+            street2: 'addressStreet2',
+            city: 'addressCity',
+            state: 'addressState',
+            zipCode: 'addressZipCode',
+            lat: 'addressLat',
+            lng: 'addressLng',
+        },
+        resultTypes = 'address',
+        shouldSaveDraft = false,
+        value,
+        locationBias,
+    }: AddressSearchProps,
+    innerRef: ForwardedRef<HTMLElement>,
```

Comment:
NAB you can rename to just ref:
```suggestion
    ref: ForwardedRef<HTMLElement>,
```